moviebot/controller/controller_flask.py

```diff
@@ -0,0 +1,398 @@
+"""This file contains the Controller class which controls the flow of the
+conversation while the user interacts with the agent using Messenger."""
+
+import json
+import os
+import sqlite3
+from datetime import datetime
+from typing import Any, Callable, Dict, Union
+
+from moviebot.agent.agent import Agent
+from moviebot.controller.controller import Controller
+from moviebot.controller.http_data_formatter import (
+    HTTP_OBJECT_MESSAGE,
+    HTTPDataFormatter,
+)
+from moviebot.core.utterance.utterance import UserUtterance
+
+
+class ControllerFlask(Controller):
+    def __init__(self) -> None:
+        """Initializes structs for Controller and sends the get started button
+        to the client."""
+        self.token = ""
+        self.agent = {}
```

Comment:
Maybe annotate some of these so it's easier to develop, use and understand. E.g., `self.agent: Dict[str, Agent] = {}`